local-kubernetes/charts/helmfile-wococo.yaml

```diff
@@ -14,15 +14,14 @@ releases:
     values:
       - ./values-testnet-manager.yml
       - configmap:
-          WS_ENDPOINT: "ws://localwococo-bootnode.wococo:9944"
-          NODE_HTTP_PATTERN: "http://NODE_NAME.wococo:9933"
-          NODE_WS_PATTERN: "ws://NODE_NAME.wococo:9944"
```

Comment:
can you also add backward compatibility for `NODE_HTTP_PATTERN` and `NODE_HTTP_PATTERN` ?